CheckDrive.Mobile/CheckDrive.Mobile/CheckDrive.Mobile/ViewModels/ProfileViewModel.cs

```diff
@@ -71,6 +78,33 @@ private async Task OnEditProfileAsync()
         {
             await _navigationService.NavigateToAsync(NavigationPageType.EditProfile);
         }
+        private async Task SaveEditsAsync()
```

Comment:
this method should be wrapped into `try-catch` block and handle any errors that occurred.